sqlite/update_statement.go

```diff
@@ -76,3 +85,11 @@ func (u *updateStatementImpl) RETURNING(projections ...Projection) UpdateStateme
 	u.Returning.ProjectionList = projections
 	return u
 }
+
+func (u *updateStatementImpl) LIMIT(limit int64) UpdateStatement {
+	if len(u.From.Tables) > 1 {
+		panic("jet: SQLite does not support LIMIT with multi-table UPDATE statements")
```

Comment:
How can we verify this? I haven't find any golang sqlite driver that compiles sqlite with SQLITE_ENABLE_UPDATE_DELETE_LIMIT.

Reply:
I'd rather not include this feature for SQLite since we can't test it, and no one would be able to use it.

Reply:
Cool, i'll drop this and raise the PR

Reply:
Done